packages/pwa-kit-build/bin/mrtcli.js

```diff
@@ -109,7 +109,36 @@ const main = () => {
                 'a custom target to upload a bundle to within Managed Runtime'
             )
         )
-        .action(({buildDirectory, message, projectSlug, target}) => {
+        .addOption(
+            new program.Option(
+                '-o --ssrOnly <ssrOnly>',
+                'the files gets uploaded to Managed Runtime and will not be accessible for the public. A list of file glob patterns separated by comma, i.e. --ssrOnly ssr.js,node_modules/*'
+            )
+        )
+        .addOption(
+            new program.Option(
+                '-s --ssrShared <ssrShared>',
+                'the files gets uploaded to Managed Runtime and will be accessible for the public to download. A list of file glob patterns separated by comma, i.e. --ssrShared static/*,**/*.js'
+            )
+        )
+        .addOption(
+            new program.Option(
+                '-f --ssrFunctionNodeVersion <ssrFunctionNodeVersion>',
+                'the Node.js version of your Managed Runtime serverless function. Availiable options are: 12.x and 14.x. Default: 14.x'
+            ).choices(['12.x', '14.x']).default('14.x')
+        )
```

Comment:
These options will all work. But I'm surprised to see them as CLI args. Didn't you want to make this the foundation of a unified config file? Create the file and put these values in there!